packages/protocol/contracts/L1/gov/TaikoGovernor.sol

```diff
@@ -58,33 +55,6 @@ contract TaikoGovernor is
         return super.propose(_targets, _values, _calldatas, _description);
     }
 
-    /// @notice An overwrite of GovernorCompatibilityBravoUpgradeable's propose() as that one does
-    /// not check that the length of signatures equal the calldata.
-    /// @dev See vulnerability description here:
-    /// https://github.com/taikoxyz/taiko-mono/security/dependabot/114
-    /// See fix in OZ 4.8.3 here (URL broken down for readability):
-    /// https://github.com/OpenZeppelin/openzeppelin-contracts/blob/
-    /// 0a25c1940ca220686588c4af3ec526f725fe2582/contracts/governance/compatibility/GovernorCompatibilityBravo.sol#L72
-    /// See {GovernorCompatibilityBravoUpgradeable-propose}
-    function propose(
-        address[] memory _targets,
-        uint256[] memory _values,
-        string[] memory _signatures,
-        bytes[] memory _calldatas,
-        string memory _description
-    )
-        public
-        virtual
-        override(GovernorCompatibilityBravoUpgradeable)
-        returns (uint256)
-    {
-        if (_signatures.length != _calldatas.length) revert TG_INVALID_SIGNATURES_LENGTH();
-
-        return GovernorCompatibilityBravoUpgradeable.propose(
-            _targets, _values, _signatures, _calldatas, _description
-        );
-    }
-
```

Comment:
This is removed because with the new version it is already patched, we no need to make this patch here.